pygmt/datasets/earth_relief.py

```diff
@@ -4,29 +4,37 @@
 
 The grids are available in various resolutions.
 """
-from pygmt.exceptions import GMTInvalidInput
+from packaging.version import Version
+from pygmt.clib import Session
+from pygmt.exceptions import GMTInvalidInput, GMTVersionError
 from pygmt.helpers import kwargs_to_strings
 from pygmt.io import load_dataarray
 from pygmt.src import grdcut, which
 
 
 @kwargs_to_strings(region="sequence")
-def load_earth_relief(resolution="01d", region=None, registration=None, use_srtm=False):
+def load_earth_relief(
+    resolution="01d",
+    region=None,
+    registration=None,
+    use_srtm=False,
+    data_source="igpp",
+):
     r"""
     Load Earth relief grids (topography and bathymetry) in various resolutions.
 
     The grids are downloaded to a user data directory
-    (usually ``~/.gmt/server/earth/earth_relief/``) the first time you invoke
-    this function. Afterwards, it will load the grid from the data directory.
-    So you'll need an internet connection the first time around.
+    (usually a subdirectory under ~/.gmt/server/).
```

Comment:
Maybe keep the note that an internet connection is needed for the first download.
```suggestion
    (usually a subdirectory under ~/.gmt/server/earth/) the first time you
    invoke this function. Afterwards, it will load the grid from the data
    directory. So you'll need an internet connection the first time around.
```